alembic.ini

```diff
@@ -0,0 +1,74 @@
+# A generic, single database configuration.
```

Comment:
Do we need this file? I think this wasn't generated by flask migrate cc @jabbate19